rfcs/058-relevance-testing/examples.md

```diff
@@ -0,0 +1,101 @@
+# Examples of rank CLI usage
+
+All flags in the following commands should be optional, and the CLI should prompt the user for any missing information, allowing them to select from the available options where appropriate.
+
+```
+rank
+├── index
+│   ├── list
+│   ├── create
+│   ├── update
+│   ├── delete
+│   ├── get
+│   └── ccr
+├── task
+│   ├── check
+│   └── delete
+├── search
+│   ├── terms
+│   └── compare
+└── test
+    ├── run
+    └── list
+```
+
+```bash
+rank index list
+```
+
+List all the indices in the rank cluster.
+
+```bash
+rank index create --source <source> --target <target> --config <config>
```

Comment:
`<source>` should be the name of the already-CCR'd source index in the rank cluster. 

Creating a new test index _from scratch_ would involve:
1. CCRing an index from a prod cluster into the rank cluster 
2. running a reindex in the rank cluster from the CCR'd index into a fresh index, with new settings/mappings

IMO we should be CCRing as little as possible, because it hammers the prod clusters. A CCR'd copy of a source index usually exists in the rank cluster already, so creating a new experimental index should only involve a reindex within rank. Keeping the processes separate (and confining the `create` command to indexes in the rank cluster) felt sensible to me.